docs/monitoring/MonitoringMigrationDashboardSharded.md

```diff
@@ -0,0 +1,88 @@
+| Worker Memory Utilization | Shows the Memory Utilization of dataflow for a shard| - 50th percentile shard Memory Utilization<br> - 90th percentile shard Memory Utilization<br> - Max percentile shard v Utilization<br> | Used to identify if the health of the pipeline for any shard based on the value of Memory Utilization| 
+| Worker Max Backlog Seconds | Shows max time required to consume the largest backlog across all stages for shards | Max | Used to identify if the pipelines is over or under scaled| 
+| Per Shard Median CPU Utilization | Shows median CPU Utilization for each shard | Total | Used to identify if any shard is struggling and the pipeline for it is underscaled| 
+
+
+### Datastream Metrics
+
+| Metric | Description | Aggregation | Relevance |
+|-----------|-----------|-----------|-----------|
+| Throughput(events/sec) | Shows the total of average events processed/sec by each shard which are generated at source | Sum | Used to track if data is being transferred from source to GCS Bucket| 
+| Unsupported Events | Total source events unsupported by Datastream | Sum | Used to identify if there is any data that can't be transferred by datastream due to a correctness issue| 
+| Total Latency | Time taken by a event in a shard for being written at source to being written by Datastream to GCS | - 50th Percentile shard<br> - 90th Percentile shaard<br> | Indicator of datastream being overloaded | 
+
+
+### GCS Bucket Metrics
+
+| Metric | Description | Aggregation | Relevance |
+|-----------|-----------|-----------|-----------|
+| Total Bytes | Shows the total bytes written to the GCS Buckets of all shards| Sum | Used to check if data is succesfully being written to the GCS Bucket| 
+
+
+### Pubsub Metrics
+
+| Metric | Description | Aggregation | Relevance |
+|-----------|-----------|-----------|-----------|
+| Published message count | Number of messages published by the GCS bucket to the Pub for all shards | Sum | Indicates total files in staging which need to be processed | 
+| Age of Oldest Unacknowledged Message | Age of the oldest unacknowledged message in the subscription in any shard| Max | Used to determine if starvation of dataflow resources is taking place |
+
+
+### Spanner Metrics
+
+| Metric | Description | Aggregation | Relevance |
+|-----------|-----------|-----------|-----------|
+| CPU Utilization | CPU Utilization of spanner database and instance| - Database Total CPU Utilization<br> - Instance Total CPU Utilization<br> | Used to track if spanner is overloaded and requires more or less nodes | 
+| Storage | Storage of spanner database and instance | - Database Total Storage<br> - Instance Total Storage<br> | Used to track how the data is growing as the migration proceeds |
+
+### Shards to Dashboard
+At the end of the Aggregated Monitoring dashboard a list of individual monitoring dashboard for each shard in a sharded migration can be found.
```

Comment:
This should be higher up, maybe below `Where is my dashboard`

Reply:
So this is mentioned on the top as well. Let me hyperlink it though.